src/network/get-info.ts

```diff
@@ -4,44 +4,29 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import { filter } from 'lodash';
 import { SHELL_APP_ID } from '../constants';
-import { useAppStore } from '../store/app';
 import { normalizeAccount } from '../store/account/normalization';
-import { GetInfoResponse, CarbonioModule } from '../../types';
+import { GetInfoResponse } from '../../types';
 import { getSoapFetch } from './fetch';
 import { useAccountStore } from '../store/account';
 import { useNetworkStore } from '../store/network';
 import { parsePollingInterval } from '../store/network/utils';
 
 export const getInfo = (): Promise<void> =>
-	fetch('/static/iris/components.json')
-		.then((r) => r.json())
-		.then(({ components }: { components: Array<CarbonioModule> }) => {
-			useAppStore
-				.getState()
-				.setters.addApps(filter(components, ({ type }) => type === 'shell' || type === 'carbonio'));
-		})
-		.then(() =>
-			getSoapFetch(SHELL_APP_ID)<{ _jsns: string; rights: string }, GetInfoResponse>('GetInfo', {
-				_jsns: 'urn:zimbraAccount',
-				rights: 'sendAs,sendAsDistList,viewFreeBusy,sendOnBehalfOf,sendOnBehalfOfDistList'
-			})
-		)
-		.then((res: GetInfoResponse): void => {
-			if (res) {
-				const { account, settings, version } = normalizeAccount(res);
-				useNetworkStore.setState({
-					pollingInterval: parsePollingInterval(settings)
-				});
-				useAccountStore.setState({
-					authenticated: true,
-					account,
-					settings,
-					zimbraVersion: version
-				});
-			}
-		})
-		.catch((err: Error) => {
-			console.error(err);
-		});
+	getSoapFetch(SHELL_APP_ID)<{ _jsns: string; rights: string }, GetInfoResponse>('GetInfo', {
```

Comment:
`getSoapFetch` suffers of the same problem of the `getComponents` fetch.
It does not throw if the response has a status different from 200-299 and the body contains a json like 
```json
{
  "Body": {}
}
```

We might consider changing how the getSoapFetch works too